From stable-bounces@linux.kernel.org  Thu Jun 30 13:07:17 2005
Date: Thu, 30 Jun 2005 16:06:41 -0400
From: Michael Krufky <mkrufky@m1k.net>
To: stable@kernel.org
Subject: [PATCH] v4l cx88 hue offset fix

Changed hue offset to 128 to correct behavior in cx88 cards.  Previously, 
setting 0% or 100% hue was required to avoid blue/green people on screen.  
Now, 50% Hue means no offset, just like bt878 stuff.

Signed-off-by: Michael Krufky <mkrufky@m1k.net>
Signed-off-by: Mauro Carvalho Chehab <mchehab@brturbo.com.br>
Signed-off-by: Chris Wright <chrisw@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/media/video/cx88/cx88-video.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

--- linux-2.6.12.2.orig/drivers/media/video/cx88/cx88-video.c	2005-06-17 12:48:29.000000000 -0700
+++ linux-2.6.12.2/drivers/media/video/cx88/cx88-video.c	2005-07-13 10:56:14.000000000 -0700
@@ -261,7 +261,7 @@
 			.default_value = 0,
 			.type          = V4L2_CTRL_TYPE_INTEGER,
 		},
-		.off                   = 0,
+		.off                   = 128,
 		.reg                   = MO_HUE,
 		.mask                  = 0x00ff,
 		.shift                 = 0,
